scenarios/azure/compute/vm.go

```diff
@@ -62,10 +62,17 @@ func NewVM(e azure.Environment, name string, params ...VMOption) (*remote.Host,
 	})
 }
 
-func defaultVMArgs(vmArgs *vmArgs) error {
+func defaultVMArgs(e azure.Environment, vmArgs *vmArgs) error {
 	if vmArgs.osInfo == nil {
 		vmArgs.osInfo = &os.UbuntuDefault
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I would switch default for azure to Windows as I believe it will be the most used one, what do you think?

Reply:
ah that makes sense!